assets/queries/cloudFormation/emr_cluster_without_security_configuration/query.rego

```diff
@@ -0,0 +1,41 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::EMR::Cluster"
+  properties := resource.Properties
+  
+  exists_security_configuration := object.get(properties, "SecurityConfiguration", "undefined") != "undefined"
+  not exists_security_configuration
+  
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s.Properties.SecurityConfiguration", [name]),
+                "issueType":		   "MissingAttribute",
+                "keyExpectedValue": sprintf("Resources.%s.Properties.SecurityConfiguration is defined", [name]),
+                "keyActualValue": 	sprintf("Resources.%s.Properties.SecurityConfiguration is undefined", [name])
+              }
+}
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::EMR::SecurityConfiguration"
+  
+  not settings_are_equal(document[i].Resources, name)
+
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s", [name]),
```

Comment:
This searchKey should point to the cluster not the securityconfiguration, the problem is when the cluster does not have a securityconfiguration